test/src/main/java/org/apache/accumulo/test/shell/ShellIT.java

```diff
@@ -557,6 +560,42 @@ public void execFileTest() throws IOException {
     assertGoodExit("Unknown command", false);
   }
 
+  @TempDir
+  private static File tempDir;
+
+  @Test
+  public void propFileNotFoundTest() throws IOException {
+    String fileName = new File(tempDir, "propFile.shellit").getAbsolutePath();
+
+    Shell.log.debug("Starting prop file not found test --------------------------");
+    exec("config --propFile " + fileName, false,
+        "FileNotFoundException: " + fileName + "(No such file or directory)");
+  }
+
+  @Test
+  public void setpropsViaFile() throws Exception {
+
+    File file = File.createTempFile("propFile", ".conf",
+        new File("src/main/resources/org/apache/accumulo/test/"));
+    PrintWriter writer = new PrintWriter(file.getAbsolutePath());
+    writer.println(COMPACTION_WARN_TIME.getKey() + "=11m");
+    writer.close();
+    exec("config --propFile " + file.getAbsolutePath(), true);
+    file.deleteOnExit();
```

Comment:
Once this test uses TempDir this line is no longer needed as tempDir is cleaned up by JUnit after a test run. 
```suggestion
```